base_layer/core/src/blocks/blockheader.rs

```diff
@@ -72,6 +74,10 @@ pub enum BlockHeaderValidationError {
     InvalidTimestampFutureTimeLimit,
     // Invalid Proof of work for the header
     ProofOfWorkError(PowError),
+    // Invalid accounting balance
```

Comment:
```suggestion
    // The total accumulated (offset) excess plus the sum of all UTXOs does not equal the total expected supply
```

Reply:
Not sure if this should be in BlockHeaderError or one level up (You do access the UTXO set, so it's not a pure header calc)

Reply:
I agree. The `InvalidAccountingBalance` error was moved to `ValidationError`.

Reply:
I changed the description to "The total expected supply plus the total accumulated (offset) excess does not equal the sum of all UTXO commitments."